### PR TITLE
Wazo 2223 mix monitor cel event

### PR DESCRIPTION
**Pull request by pc-m**
Depends-On: https://github.com/wazo-platform/asterisk/pull/62

### CODE REVIEW

**Comment by fblackburn1** [comment]
recheck

**Comment by fblackburn1** [comment]
recheck

**Comment by fblackburn1** [comment]
recheck

**Comment by fblackburn1** [comment]
@pc-m any idea why it fails to find the downloaded tarball? :thinking:

**Comment by pc-m** [comment]
> @pc-m any idea why it fails to find the downloaded tarball?

no clue, will retry today

**Comment by pc-m** [comment]
recheck

File: debian/patches/wazo_mixmonitor_events [hunk 1]
```diff
@@ -11,51 +11,81 @@ Index: asterisk-18.2.0/apps/app_mixmonitor.c
...
++	RAII_VAR(struct ast_json *, blob, NULL, ast_json_unref);
  
  	AST_DECLARE_APP_ARGS(args,
  		AST_APP_ARG(mixmonid);
-@@ -1263,6 +1274,13 @@ static int stop_mixmonitor_full(struct a
+@@ -1229,6 +1246,12 @@ static int stop_mixmonitor_full(struct a
+ 
+ 	ast_mutex_lock(&mixmonitor_ds->lock);
+ 
++	if (ast_asprintf(&datastore_id, "%p", mixmonitor_ds) == -1) {
```
Comment (sduthil) [inline debian/patches/wazo_mixmonitor_events:61]: What do you think of using `args.mixmonid` instead?

**Review by sduthil — CHANGES_REQUESTED** [review]

File: debian/patches/wazo_mixmonitor_events (hunk 1 above)
Comment (pc-m) [reply debian/patches/wazo_mixmonitor_events:61]: args.mixmonid is only valid when it has been specified on the command line

Comment (sduthil) [reply debian/patches/wazo_mixmonitor_events:61]: OK, thanks!

**Review by sduthil — APPROVED** [review]

**Review by fblackburn1 — APPROVED** [review]
Approve functional

**Pull request merged**